test.js

```diff
@@ -204,7 +204,7 @@ test('assign styles', t => {
 });
 
 test('assign other props', t => {
-	const el = <a href="video.mp4" id="a" download referrerPolicy="no-referrer">Download</a>;
```

Comment:
I'd like to keep the props React-like and camelCased. Is there a list of props that must remain camel cased, so that we could skip `toLowerCase()` on them only? Does this apply only to SVG?